### ISSUE TITLE
🐛 param decorators are flagged as invalid even with unsafeParameterDecoratorsEnabled: true

### ISSUE THREAD

**Issue by RobbyUitbeijerse** [biomejs/biome#145]
### Environment information

```block
CLI:
  Version:                      1.0.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v18.17.1"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "pnpm/8.7.0"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Workspace:
  Open Documents:               0

Discovering running Biome servers...

Incompatible Biome Server: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

ℹ Rage discovered this running server using an incompatible version of Biome.

Server:
  Version:                      12.1.3

Incompatible Biome Server: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

ℹ Rage discovered this running server using an incompatible version of Biome.

Server:
  Version:                      <=10.0.0

Incompatible Biome Server: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

ℹ Rage discovered this running server using an incompatible version of Biome.

Server:
  Version:                      12.1.3-nightly.4c8cf32
```

Hello! Congrats on the name change. 

This issue in the rome repository is related:
https://github.com/rome/tools/issues/4562

### What happened?

1. Apply param decorators (I'm using nestjs myself)
2. Apply javascript.parser.unsafeParameterDecoratorsEnabled = true in the biome.json
3. Observe how errors persist 

package.json:

```
  "dependencies": {
    "@biomejs/biome": "^1.0.0"
  },
  ```

The rage output is a bit confusing. I've re-installed all the deps within the project the complaint about an invalid biome server that is running somewhere persists. Might be good to know that I have other projects on my pc that are still running rome 12.1.3, but I'm not sure if that could be considered related.

Here are some screenshots:

![Screenshot 2023-09-04 at 14 12 35](https://github.com/biomejs/biome/assets/10176709/d6b7fc0b-4451-4340-9f69-f60ae14117aa)
![Screenshot 2023-09-04 at 14 12 37](https://github.com/biomejs/biome/assets/10176709/263fcedf-c4ce-43ee-b3cb-68062dbc7a70)

### Expected result

No errors being displayed in vscode.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Are you experiencing the same issue via CLI?

**Comment by ematipico** [comment]
From your rage, it seems you have multiple server running. Could you try to kill all the services? `killall biome` and `killall rome`. It's possible you have an old server running that shows incorrect diagnostics. 

I tested locally and I can't reproduce the issue.

**Comment by woifes** [comment]
@ematipico  can you please give more detail how to stop the processes?
